avs/utils/contract-data-gen.sh

```
@@ -0,0 +1,13 @@
+mkdir -p src/avs/contracts
+typechain -- --target ethers-v5 --out-dir src/avs/contracts \
+	avs/contracts/out/ReclaimServiceManager.sol/*.json \
+	avs/contracts/out/ReclaimTaskManager.sol/*.json \
+	avs/contracts/out/ReclaimSlashingRegistryCoordinator.sol/*.json \
+	avs/contracts/out/AllocationManager.sol/*.json \
+	avs/contracts/out/AVSDirectory.sol/*.json \
+	avs/contracts/out/DelegationManager.sol/*.json \
+	avs/contracts/out/ECDSAStakeRegistry.sol/*.json \
+	avs/contracts/out/ERC20Mock.sol/*.json \
+	avs/contracts/out/RewardsCoordinator.sol/*.json \
+	avs/contracts/out/SocketRegistry.sol/*.json \
```

Shellcheck (0.10.0)

[error] Couldn't parse this escaped char. Fix to allow more checks.

(SC1073)